packages/TorneloScoresheet/src/types/AppModeState.ts

```diff
@@ -7,14 +7,20 @@ export enum AppMode {
   ResultDisplay,
 }
 
+export enum ArbiterModeViews {
+  EnterPgnLink,
+  TablePairingSelection
+}
 export type ArbiterSetupMode = {
   mode: AppMode.ArbiterSetup;
+  view: ArbiterModeViews
+  games?: number;
+  pairings?: GameInfo[];
```

Comment:
I'm thinking: rather than having this one mode have two views, why don't we just invent a new mode that happens before arbiter setup mode? Like PGNDownload mode or something (we can come up with a better mode). I think this would make the state a bit more easy to understand? This also works though - just a though, let me know what you think

Reply:
Yeh will make a lot more sense to have 1 view per state, i changed it now, how should we handle the state transition fuctions though, its gonna get pretty messy once we have a lot of transitions from different views, (idk about my naming convention too, the function names are a little bit long)

Reply:
Hmm yeah. I think long names are fine tbh - probably more important to make it clear what the function is doing then save characters imo. I also *think* the state transition stuff will be fine since it will all be defined in code that is completely decoupled from the views. It might be worth considering moving some of this stuff out of the global state and into local state tbh. Let's discuss this further in the sprint retro and come up with a strategy